CritterServer/Pipeline/ErrorMiddleware.cs

```
@@ -27,6 +28,12 @@ public async Task InvokeAsync(HttpContext context)
             {
                 await next.Invoke(context);
             }
+            catch(CritterException cex) 
+            {
+                Log.Information(cex, cex.InternalMessage);
```

If all of them are funneling into this single catch block, a way of specifying the log level when throwing the exception might allow for a bit more control rather than everything ending up as Information

---

My thinking here was that this represents a functional exception, something like when validation fails and we want to return an error to the user, but that isn't something we'd likely care to see in the logs. Maybe add a log level override though.